### PR TITLE
Section page

### PR DESCRIPTION
**Pull request by jowlee**
Refactored section page for instructor.

### CODE REVIEW

**Comment by samanthajlee2** [comment]
now realizing that for longer course names, this doesn't work very well...

Do we have a character limit for course names?
Maybe this needs to be a two liner...
Course Name
Dept. #### Sections #,#...

may11_185837

**Comment by samanthajlee2** [comment]
Also, can we swap the Events and Students tabs such that we have
| Info | Students | Events | Submissions |

may11_190143

File: client/app/courses/sections/view/sectionview.controller.js [hunk 1]
```diff
@@ -15,6 +15,8 @@ angular.module('venueApp')
       }, section => {
         $scope.course = section.course;
         $scope.section = section;
+        // TODO: automate by first index of events for Section.
+        $scope.loadSectionEventSubmissions(section.events[0].sectionEvents[0]);
```
Comment (seveibar) [inline client/app/courses/sections/view/sectionview.controller.js]: remove dis

File: client/app/instructor/courses/sections/view/sectionview.controller.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        withSectionsEvent: true,
+        withSectionsInstructors: true,
+        withSectionsStudents: true,
+        withSectionsPendingStudents: true,
+        withEnrollmentStatus: true,
+        studentId: $scope.user._id
+      }, section => {
+        $scope.course = section.course;
+        $scope.section = section;
+        // TODO: automate by first index of events for Section.
```
Comment (seveibar) [inline client/app/instructor/courses/sections/view/sectionview.controller.js]: remove dis

File: client/app/instructor/courses/sections/view/sectionview.controller.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    $scope.editCourse = function(){
+      $location.path('/courses/'+$routeParams.id+'/edit');
+    };
+
+    $scope.viewCourse = function(){
+      $location.path('/courses/'+$routeParams.id);
+    };
+
+    $scope.editSection = function(){
+      console.log('/courses/' + $routeParams.id + '/sections/' + $routeParams.sectionId + '/edit');
```
Comment (seveibar) [inline client/app/instructor/courses/sections/view/sectionview.controller.js]: remove dis

File: client/app/instructor/courses/sections/view/sectionview.html [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+              <span class="big-number text-right">
+                <span ng-repeat="number in section.sectionNumbers track by $index" ng-show="number > -1">
+                  <span ng-show="$last && !$first">& </span>{{number}}<span ng-hide="$last || section.sectionNumbers.length < 3">, </span>
+                </span>
+            </div>
+            <button class="btn btn-default edit-Section col-sm-2" ng-click='editSection()'>Edit</button>
+          </div>
+        </a>
+      </div>
+    </div> -->
```
Comment (seveibar) [inline client/app/instructor/courses/sections/view/sectionview.html]: spooky

File: client/app/instructor/courses/sections/view/sectionview.html [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+              <div class="col-sm-12">
+                <h4>Submission Preview</h4>
+                <!-- <uib-carousel>
+                  <uib-slide ng-repeat="student in section.students" active="slide.active">
+                    <img http-src="{{student.submission.images[0]}}" alt="submission-image" class="materialboxed" style="height: 200px; width: 600px; overflow: hidden; object-fit: contain;"/>
+                    <div class="carousel-caption">
+                      <h5>{{student.firstName}} {{student.lastName}}</h5>
+                    </div>
+                  </uib-slide>
+                </uib-carousel> -->
```
Comment (seveibar) [inline client/app/instructor/courses/sections/view/sectionview.html]: scary

File: server/api/submission/submission.controller.js [hunk 6]
```diff
@@ -196,6 +196,7 @@ exports.index = function(req, res) {
         .catch(handleError(res));
     }
     else{
+      console.log(req.query.onlySectionEvent);
```
Comment (seveibar) [inline server/api/submission/submission.controller.js]: log loggy log

**Pull request merged**